### PR TITLE
operator silicom-sts-operator (0.0.6)

### PR DESCRIPTION
**Pull request by rmr-silicom**
Signed-off-by: rmr <rmr@silicom.dk>

### CODE REVIEW

File: operators/silicom-sts-operator/0.0.6/manifests/silicom-sts-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+                    env:
+                      - name: GPS_SVC_PORT
+                        value: "2947"
+                      - name: GRPC_SVC_PORT
+                        value: "50051"
+                      - name: NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
+                    image: quay.io/silicom/sts-operator@sha256:a79a5cf99f5238ee08c509efe0b6142197ef0b79a6b209c125d9f7b7235758e5)
```
Comment (acornett21) [inline operators/silicom-sts-operator/0.0.6/manifests/silicom-sts-operator.clusterserviceversion.yaml:481]:
```suggestion
                    image: quay.io/silicom/sts-operator@sha256:a79a5cf99f5238ee08c509efe0b6142197ef0b79a6b209c125d9f7b7235758e5
```

Comment (rmr-silicom) [reply operators/silicom-sts-operator/0.0.6/manifests/silicom-sts-operator.clusterserviceversion.yaml]: Fixed. Thx

**Pull request closed without merging**